release/docker/v6_deb.dockerfile

```diff
@@ -6,13 +6,15 @@ ARG EXTENSION
 ARG TARGETARCH
 
 RUN apt-get update && apt-get install -y \
-  openssl libcurl4 libssl3 libseccomp2 python3 libpython3.12 python3-pip libatomic1 adduser \
-  --no-install-recommends \
-  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+  openssl libcurl4 libssl3 libseccomp2 python3 libpython3.12 python3-pip libatomic1 adduser libxmlsec1-dev xmlsec1 \
+  --no-install-recommends 
 
 # Bugfix for timezone issues - pin the package
-RUN apt install tzdata=2024a-2ubuntu1
-RUN sudo apt-mark hold tzdata
+RUN apt install -y tzdata=2024a-2ubuntu1 --allow-downgrades
+RUN apt-mark hold tzdata
+
+# do this after apt, not before!
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
This should be moved together into the `apt install` line, we already talked about it, just to have a mention here 😃